### ISSUE TITLE
[BUG/GLITCH] Cast Members not appearing when clicking Context Menu, Information from titles under the Information Tab

### ISSUE THREAD

**Issue by Nuklear92** [nfm886/skin.arctic.zephyr.2.resurrection.mod#4]
Hello, so I decided to open and follow up issues I find here on Github instead. 

So, this is another issue I have found which is plaguing the Information Tab along with the other's I've mentioned you through Reddit. This one is being triggered specially when you're checking out any of the recommended, similar, from studio or any other of the given lists under the Cast Members. Whenever you click on any of those titles and open the context menu followed by Information as you'll see on a small clip I recored and shared that once you open it Cast Members along side some of the list are missing or simply not appearing due to some sort of glitch. By the way this is of course on the latest build of AZ2R you pushed 0.3.3. 

https://imgur.com/a/CvYOsm2

**Comment by nfm886** [comment]
Thanks for your feedback!
I will investigate the issue. There is some work to do with DialogVideoInfo and themoviedb helper to improve performance a bit. Maybe then I will try to fix some other bugs.

**Comment by Nuklear92** [comment]
> Thanks for your feedback!
> 
> I will investigate the issue. There is some work to do with DialogVideoInfo and themoviedb helper to improve performance a bit. Maybe then I will try to fix some other bugs.

Thanks, I'll be sure to report any new bugs I find or feature requests through here better. 😃

**Comment by Nuklear92** [comment]
@nfm886 Heres the clip I promised about trailers not playing in the latest build, still. As you can see in order to reproduce it, you need to click on any of the titles present under the information tab as part of the given lists of; recommended, similar, by director, etc. once you click on the title it then opens the title within the tmdbhelper addon right in the information tab and once you click on the trailer it pretty much stays frozen and nothing plays. Hence this issue only happens when clicking on the titles via the Information Tab. If you go and open the tmdbhelper addon separately and look for the same movie/tv show title and click on the information tab and trailer. It works as usual, but the whole point of clicking on titles via the information tab and be able to watch the trailer, etc right away is the convenience of discovering new stuff to watch, etc when you're viewing the information of a title and under the casts you see a title recommended that you might be interested and quick opening through and being able to see the trailer etc. quickly. 

https://imgur.com/a/1vI1UM0

**Comment by nfm886** [comment]
Thanks! You are right, in that way you can discover new movies more convenience.
I hope I will be able to fix it. From what one of users says: https://github.com/nfm886/skin.arctic.zephyr.2.resurrection.mod/issues/9#issuecomment-860731505 it's partially fixed so we should be now closer to fix it than before. It taking that many time to fix since it not throwing any errors so it's so hard to investigate :/ But I will try to do my best!

**Comment by nfm886** [comment]
By the way. According to this bug: https://imgur.com/a/EUJe8CA
It seems, it fixed now: https://streamable.com/nebcqn :)

**Comment by Nuklear92** [comment]
> By the way. According to this bug: https://imgur.com/a/EUJe8CA
> 
> It seems, it fixed now: https://streamable.com/nebcqn :)

Actually, the way you're doing it in this clip has always worked. Because, if you take notice you're not click opening the title. You're actually long pressing and opening the **Context Menu** and clicking **Information** right from the little thumbnail there. Try instead, clicking the title, don't open the **Context Menu**, just click on any of the titles right from the **Information** tab and then it will open it through the TMDb Helper add-on. Then click on the trailer and you'll see how it gets frozen and doesn't play anything like you saw on the clip I shared. 

You might not noticed, but in your clip you also triggered the official bug I shared on this GitHub post. The one that when you long press and open context menu, followed by Information on and titles shown in the Information tab as part of the similar, recommended, by studio, etc. you'll see that the cast is not present it's bugged or glitched. So, these are literally two bugs that you reproduce a bit differently.

**Comment by nfm886** [comment]
Thanks for your explaining. I have some work to do with Netflix style layout and then we will think what to do with it. I hope it will be ready for today's update in the evening.

**Comment by nfm886** [comment]
I will try to close all the issues mentioned on GitHub. This will be my next goal.

**Comment by Nuklear92** [comment]
> Thanks for your explaining. I have some work to do with Netflix style layout and then we will think what to do with it. I hope it will be ready for today's update in the evening.

Sure, no rush take your time. Just wanted to better explain how to reproduce the issue and of course showing that it's present on the latest update. By the way I saw your comment about sharing the Netflix Style MediaView UIs I told you about, I'll share them in a moment. I might do it through there as well and tag you. 👍

**Comment by Nuklear92** [comment]
@nfm886 Okay, here we go. These are all the different Netflix like MediaViews available in AuraMod skin. Before anything, this is to show you so you can have an idea, in case you did like in the near future to create your own mediaview UIs to accompany the home screen Netflix like UI you're currently working on. I'm in no way, shape or form asking you to do this. Take it more of like an inspirational way.

Okay, let's start with the TV Shows different views since they have the most. I'm gonna call each view as it's called when changing media views in the AuraMod skin.

**View Mode: Showcase**
![image](https://user-images.githubusercontent.com/76958541/122068734-a6c1cb80-cdc2-11eb-9ad9-6772e5aca000.png)

**TVShows View Mode: Landscape Showcase**
![image](https://user-images.githubusercontent.com/76958541/122068978-d1ac1f80-cdc2-11eb-9b19-954282af8d79.png)

**TVShows View Mode: Panel Showcase**
![image](https://user-images.githubusercontent.com/76958541/122069125-ee485780-cdc2-11eb-92c8-b26ed9769c81.png)

**TVShows Context Menu, Information**
![image](https://user-images.githubusercontent.com/76958541/122069543-45e6c300-cdc3-11eb-947e-2d5fbf2e5782.png)

**TVShows Information, Cast and More**
![image](https://user-images.githubusercontent.com/76958541/122069710-67e04580-cdc3-11eb-84f4-1c53d5b3da7a.png)
![image](https://user-images.githubusercontent.com/76958541/122069750-6e6ebd00-cdc3-11eb-84bd-66ed2f83a187.png)

**TVShows Seasons View Mode: Netflix Season**
![image](https://user-images.githubusercontent.com/76958541/122070220-c60d2880-cdc3-11eb-8b24-25d2b03ef3f1.png)

**TVShows Seasons View Mode: Showcase**
![image](https://user-images.githubusercontent.com/76958541/122070471-ffde2f00-cdc3-11eb-9bd9-1a5922097b9e.png)

**TVShows Episodes View Mode: Landscape Showcase**
![image](https://user-images.githubusercontent.com/76958541/122071005-64998980-cdc4-11eb-8f73-36ea009c7602.png)

**TVShows Episodes View Mode: Episode List**
![image](https://user-images.githubusercontent.com/76958541/122071177-88f56600-cdc4-11eb-82dd-61ea544d9cd7.png)


**Movies View Mode: Showcase**
![image](https://user-images.githubusercontent.com/76958541/122071513-c823b700-cdc4-11eb-9725-18334f6259cf.png)

**Movies View Mode: Landscape Showcase**
![image](https://user-images.githubusercontent.com/76958541/122071584-da9df080-cdc4-11eb-8b9b-fba4b7fd9d0b.png)

**Movies View Mode: Panel Showcase**
![image](https://user-images.githubusercontent.com/76958541/122071683-f1444780-cdc4-11eb-9576-7e3b024507ac.png)

**Movies, Context Menu - Information **
![image](https://user-images.githubusercontent.com/76958541/122071872-1a64d800-cdc5-11eb-9bc9-358fe196796b.png)

**Movies, Information - Cast and More**
![image](https://user-images.githubusercontent.com/76958541/122071994-35374c80-cdc5-11eb-814b-d936bf86c8fd.png)
![image](https://user-images.githubusercontent.com/76958541/122072020-39636a00-cdc5-11eb-95f2-0a0885c1fb2a.png)
![image](https://user-images.githubusercontent.com/76958541/122072045-3e281e00-cdc5-11eb-8862-dece34fc56f2.png)

So, there it is. Of course, in case you ever would like to try one or some of those MediaViews for AZR you might want to give it your own personal touch and follow the UI design language of Arctic Zephyr so it maintains the same feeling and elegance of the original skin but with that touch of Netflix as you seen in those screenshots above. 🙂

**Comment by nfm886** [comment]
Hmm... I must say it looks cool! :)
I will think how to implement this but it's a lot of work to do ^^

**Comment by Nuklear92** [comment]
> Hmm... I must say it looks cool! :)
> 
> I will think how to implement this but it's a lot of work to do ^^
> 
> 

Yeah, I can imagine. There's absolutely no rush at all on that. I was mainly sharing the screenshots so you had an idea of the companion media views that can be possible with your already work in progress Home Screen Netflix style UI. 🙂 but keep working on your bug fixes and everything else that could be something for later on in the future when you don't have much to do left with the skin. 😄

**Comment by nfm886** [comment]
I'm kind of newbie and still learning. My skills are limited but I'm trying so hard! :)

**Comment by Nuklear92** [comment]
@nfm886 It's all good, no worries. Don't feel pressured in any way and keep working with what you can and already have knowledge. You're doing a great job so far, cleaning up some of the bugs among other things this skin had previously. 🙂👍

**Comment by nfm886** [comment]
Thanks for kind words ;)
